packages/bpk-svgs/dist/js/icons/lg/account.js

```diff
@@ -1,14 +1,8 @@
-function _objectWithoutProperties(obj, keys) { var target = {}; for (var i in obj) { if (keys.indexOf(i) >= 0) continue; if (!Object.prototype.hasOwnProperty.call(obj, i)) continue; target[i] = obj[i]; } return target; }
-
 import React from "react";
-export default ((_ref) => {
-  let {
-    styles = {}
-  } = _ref,
-      props = _objectWithoutProperties(_ref, ["styles"]);
-
-  return <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="24" height="24" style={{
-    width: "1.5rem",
-    height: "1.5rem"
-  }} {...props}><path d="M3 21v-3.1c0-2.8 2.2-5 5-5h8c2.8 0 5 2.2 5 5V21H3zM16 7c0-2.2-1.8-4-4-4S8 4.8 8 7s1.8 4 4 4 4-1.8 4-4z" /></svg>;
-});
\ No newline at end of file
+export default (({
+  styles = {},
+  ...props
+}) => <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" style={{
+  width: "1.5rem",
+  height: "1.5rem"
+}} {...props}><path d="M3 21v-3.1c0-2.8 2.2-5 5-5h8c2.8 0 5 2.2 5 5V21H3zM16 7c0-2.2-1.8-4-4-4S8 4.8 8 7s1.8 4 4 4 4-1.8 4-4z" /></svg>);
```

Comment:
Do we know why this is no longer being transpiled?

Reply:
They dropped support for Node 6 so it might be just be using more modern syntax (https://github.com/boopathi/react-svg-loader/blob/master/CHANGELOG.md#300)

It was never transpiled earlier as it was using `export` which isn't valid ES5.